bedrock/base/templates/base-resp.html

```diff
@@ -60,6 +56,7 @@
     {# Global styles, hidden from IE7 and lower #}
     {% block site_css %}
       {% stylesheet 'responsive' %}
+      {% stylesheet 'tabzilla-static' %}
```

Comment:
Yes the only css in the 'responsive' bundle is 'sandstone-resp.less', so I can definitely add it. Glad you suggested. I caught a few other places where tabzilla-static was part of page specific bundles I can remove.